### DISCUSSION TITLE
Compatibility issues while using Prophet and Holidays in Azure ML

### DISCUSSION THREAD

**Discussion by Vamsikethepalli** [vacanza/holidays#1887] [Q&A]
Hello Everyone,

I was developing a forecast model in Azure ML using Prophet and Holidays but there is some issue while creating an environment for job. The error is in below attached screenshot. 
I saw some previous posts and tried using holidays == 0.25 but it is also not working as the error is 
2024-07-30T13:23:21: ERROR: Could not find a version that satisfies the requirement azureml-mlflow (from versions: none)
2024-07-30T13:23:21: ERROR: No matching distribution found for azureml-mlflow

![Prophet1](https://github.com/user-attachments/assets/6fffd406-b347-4c61-827f-e28dcc2c074f)

Please suggest compatible versions of prophet and holidays to use in Azure Ml or if im doing anything wrong please let me know 

Thanks

**Comment by arkid15r** [comment]
Hi @Vamsikethepalli 

It seems that you should be fine with the most recent version. As far as I could found my most recent update to prophet looked like [this](https://github.com/facebook/prophet/commit/8cf3ba9e466f9c115036658198ff40f832aa6685#diff-1a45bd938160f7761769e7b6fbc819d1cc7f84ce37f1a6a23ef83cc4e214abc8L22)

It also might be the issue is not related to holidays package (see line 513).
